modules/data.land/R/helpers.R

```diff
@@ -0,0 +1,7 @@
+stop <- function(...) {
```

Comment:
I totally agree! I suggested creating a separate logger package, and more generally, making our packages smaller and more precisely focused, in #1191. We should discuss at the call on Thursday -- it's easy to implement, but would require revising a lot of existing PEcAn code and packages.